combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt

```diff
@@ -691,4 +774,11 @@ internal class NetworkManager(
             ProbeDiscoveredEvent.ProbeRemoved(serialNumber)
         )
     }
+
+    var messageTypeCallback: (UByte)-> NodeMessage? = { messageType: UByte -> NodeMessage.fromUByte(messageType) }
+        private set
+
+    internal fun setMessageTypeCallback(callback: (UByte)-> NodeMessage?) {
```

Comment:
This setter approach is fine, but  I want to point out [this](https://github.com/combustion-inc/combustion-android-ble/blob/bf38b0945b473a07ad69eb1e4bd44638fce2b8ad/combustion-android-ble/src/main/java/inc/combustion/framework/service/DeviceManager.kt#L72) structure.

We use that to pass in configurable parameters into the service on it's construction.  We also initialize the allow-list that way.  It might be cleaner to use that.  Unless you want to be able to update/change the type callback after initialization.

Reply:
I don't think there is a need to change the callback after initialization. I'll update it to use that pattern.

Reply:
Updated.